### ISSUE TITLE
Add proper text-color for hsla

### ISSUE THREAD

**Issue by matthova** [egonyans/vscode-ext-color-highlight#90]
When I use a white hsla value such as 
```
background: "hsla(0, 0%, 100%, 1)",
```
The text is white and the background is white.

**Comment by wesbos** [comment]
Yeah this is key, the background colour can wash out a lot of the text:

<img width="518" alt="image" src="https://user-images.githubusercontent.com/176013/104746184-ece40780-571c-11eb-8d5c-17f59a471e34.png">

We could easily calculate the opposite colour, or a threshold for dark/light colours. I've done something similar to that here: https://github.com/wesbos/beginner-javascript/blob/master/exercises/84%20-%20Web%20Speech%20Colour%20Game/colors.js#L152-L158

**Comment by wangyunduo** [comment]
Hi, there 👋 I notice that this repo has not had any new commits for the past year, and version 2.6.0, submitted last year, has not been released. Therefore, I forked the repo and developed a new extension based on version 2.6.0, named Color Highlight - Hue Show Better Version.

The extension [Color Highlight - Hue Show Better Version](https://marketplace.visualstudio.com/items?itemName=Yunduo.color-highlight-css-color-4) can solve your issue. Feel free to try it out : )
https://marketplace.visualstudio.com/items?itemName=Yunduo.color-highlight-css-color-4

<img width="481" alt="image" src="https://github.com/enyancc/vscode-ext-color-highlight/assets/37109016/537a1999-1506-4ff5-a592-084f98729919">
